workspaces/web/src/components/ChallengePanel.svelte

```diff
@@ -80,5 +81,10 @@
 
   .panel-block {
     color: inherit;
+    padding-top: 1em;
```

Comment:
I wonder if panel-block should rather have a fixed width instead?

Reply:
`panel-block` still has fixed width. A slight padding is added to top to push buttons or text below, to create more space above the buttons for cleaner look.